docker/Dockerfile.scope

```diff
@@ -0,0 +1,9 @@
+FROM weaveworks/cloud-agent
+RUN apk add --update runit && \
+	rm -rf /var/cache/apk/*
+ADD ./demo.json /
+COPY ./runsvinit ./entrypoint.sh /home/weave/
```

Comment:
Does `/home/weave` need to be copied again in `Dockerfile.scope` because `RUN apk add --update runit` modifies it?